stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/viewholder/decorator/Decorator.kt

```diff
@@ -3,6 +3,6 @@ package io.getstream.chat.android.ui.messages.adapter.viewholder.decorator
 import com.getstream.sdk.chat.adapter.MessageListItem
 import io.getstream.chat.android.ui.messages.adapter.BaseMessageItemViewHolder
 
-internal interface Decorator {
-    fun <T : MessageListItem> decorate(viewHolder: BaseMessageItemViewHolder<T>, data: T)
+public interface Decorator {
```

Comment:
I think we'll need to figure out how not to expose it to customers. There is no sense in decorators being public